tortoise/queryset.py

```diff
@@ -572,7 +575,11 @@ def distinct(self) -> QuerySet[MODEL]:
         return queryset
 
     def select_for_update(
-        self, nowait: bool = False, skip_locked: bool = False, of: tuple[str, ...] = ()
+        self,
```

Comment:
It isn't a blocker in any way but if you have time, could you please document the meaning of the arguments?

Reply:
Thanks for the review! I've added some documentation for the arguments.

I think the build is currently failing because it requires the pypika-tortoise changes and the test run installs v0.5.0 which doesn't have those changes yet. I've updated the version requirements in the PR, but it gets blocked and won't build until a new pypika-tortoise version is released.

Alternatively, we could add version checks to provide backwards compatibility with older versions of pypika-tortoise, but that doesn't seem very attractive to me.

Reply:
I'll release the new version of pypika to day, and will let you know! Thanks!

Reply:
I just released pypika-tortoise 0.6.0 with your changes.

Reply:
@henadzit Let's clean the PRs of pypika-tortoise and release a new version. Do you want to continue this https://github.com/tortoise/pypika-tortoise/pull/21, or let me pick it up?